edge/pkg/edged/edged.go

```diff
@@ -400,34 +372,33 @@ func getConfig() *Config {
 	}
 
 	if conf.runtimeType == "" {
-		conf.runtimeType = DockerContainerRuntime
+		conf.runtimeType = RemoteContainerRuntime
 	}
-	if conf.runtimeType == RemoteContainerRuntime {
-		conf.memoryCapacity = config.CONFIG.GetConfigurationByKey("edged.edged-memory-capacity-bytes").(int)
-		if conf.memoryCapacity == 0 {
-			conf.memoryCapacity = MinimumEdgedMemoryCapacity
-		}
-		conf.remoteRuntimeEndpoint = config.CONFIG.GetConfigurationByKey("edged.remote-runtime-endpoint").(string)
-		if conf.remoteRuntimeEndpoint == "" {
-			conf.remoteRuntimeEndpoint = RemoteRuntimeEndpoint
-		}
-		conf.remoteImageEndpoint = config.CONFIG.GetConfigurationByKey("edged.remote-image-endpoint").(string)
-		//runtimeRequestTimeout := config.CONFIG.GetConfigurationByKey("edged.runtime-request-timeout").(int)
-		//conf.RuntimeRequestTimeout = metav1.Duration{Duration: runtimeRequestTimeout * time.Minute}
-		if conf.RuntimeRequestTimeout == zeroDuration {
-			conf.RuntimeRequestTimeout = metav1.Duration{Duration: 2 * time.Minute}
-		}
-		conf.PodSandboxImage = config.CONFIG.GetConfigurationByKey("edged.podsandbox-image").(string)
-		if conf.PodSandboxImage == "" {
-			conf.PodSandboxImage = PodSandboxImage
-		}
-		if conf.cgroupDriver == "" {
-			conf.cgroupDriver = "systemd"
-		}
-		if net.ParseIP(conf.nodeIP) == nil {
-			//loopback
-			conf.nodeIP = "127.0.0.1"
-		}
+
+	conf.memoryCapacity = config.CONFIG.GetConfigurationByKey("edged.edged-memory-capacity-bytes").(int)
+	if conf.memoryCapacity == 0 {
+		conf.memoryCapacity = MinimumEdgedMemoryCapacity
+	}
+	conf.remoteRuntimeEndpoint = config.CONFIG.GetConfigurationByKey("edged.remote-runtime-endpoint").(string)
+	if conf.remoteRuntimeEndpoint == "" {
+		conf.remoteRuntimeEndpoint = RemoteRuntimeEndpoint
+	}
+	conf.remoteImageEndpoint = config.CONFIG.GetConfigurationByKey("edged.remote-image-endpoint").(string)
+	//runtimeRequestTimeout := config.CONFIG.GetConfigurationByKey("edged.runtime-request-timeout").(int)
+	//conf.RuntimeRequestTimeout = metav1.Duration{Duration: runtimeRequestTimeout * time.Minute}
+	if conf.RuntimeRequestTimeout == zeroDuration {
+		conf.RuntimeRequestTimeout = metav1.Duration{Duration: 2 * time.Minute}
+	}
+	conf.PodSandboxImage = config.CONFIG.GetConfigurationByKey("edged.podsandbox-image").(string)
+	if conf.PodSandboxImage == "" {
+		conf.PodSandboxImage = PodSandboxImage
+	}
+	if conf.cgroupDriver == "" {
```

Comment:
anywhere set the conf.cgroupDriver and conf.nodeIP from config file?

Reply:
https://github.com/kubeedge/kubeedge/blob/de2e665ff16aaa8128833dfea376bf8f0befda67/edge/conf/edge.yaml#L53-L54

Reply:
how about merge these line to `config.CONFIG.GetConfigurationByKey`, if !ok, just set this default value

Reply:
done